_unittest/example_models/report_json/EyeDiagram_Report.toml

```
@@ -0,0 +1,117 @@
+Help = "This Report Template is applicable to Eye Diagram Plots from Transient and Eye Simulations. It include Mask section."
```

```suggestion
Help = "This report template is applicable to eye diagram plots from transient and eye simulations. It includes a mask section."
```
Would we please use sentence case for improved readability. Using initial capital letters on lots of words greatly reduces readability.